draft.md

```diff
@@ -0,0 +1,101 @@
+# Draft IPLD Unixfs Spec
```

Comment:
Let's stop calling this IPLD Unixfs, **the current Unixfs is already IPLD**. This proposal is to:

- Move away from dag-pb to dag-cbor
- Leverage the flexibility of dag-cbor to add more Metadata
- Improve Unixfs and remove some of the limitations we found while using unixfs with dag-pb.

One of the design goals of the new Unixfs is that it should be 100% interopable with the old (a directory of Unixfs2 should be able to have a file of Unixfs1)

Reply:
The name "IPLD Unixfs" was @whyrusleeping idea and I just went along with it.  We could call it "Unixfs V2", although I am not sure how much with want to stick with the unix filesystem structure as a model (I personally thing we should move away from it and focus on the compartments that are important to a generic archive structure).

Reply:
I'm fine not calling it ipld unixfs. @diasdavid is right

Reply:
>  a directory of Unixfs2 should be able to have a file of Unixfs1

It's probably worth mentioning this in the spec.